Source/LocationManager.swift

```diff
@@ -18,6 +18,14 @@ open class LocationManager {
             self = .coreLocation
         }
     }
+    open func authorizationStatus() -> CLAuthorizationStatus {
```

Comment:
Adding this as a class method means that to use LocationManager you need to switch over calls to authorizationStatus to the instance. This worked for my use-case, but it isn't ideal.